### PR TITLE
Feature deploy client

### PR DESCRIPTION
**Pull request by fedescarpa**

### CODE REVIEW

File: deploy-client.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+cd client
+echo "Moved to: ${PWD}"
+
+if [ "$TRAVIS_BRANCH" = "master" ] && [ "$TRAVIS_PULL_REQUEST" = "false" ]; then
```
Comment (flbulgarelli) [inline deploy-client.sh]: This is script is similar to that in editor, right?

Could we code it just once and wget it here?

**Comment by flbulgarelli** [comment]
Is this already mergeable @fedescarpa ?

**Comment by fedescarpa** [comment]
I think so :smile:

**Comment by flbulgarelli** [comment]
Looks good to me. 

I still think we could make the editor build similar to this build, so that we can reuse the deployment scripts, at least partially. 

And that means we would expect the same JS configuration regarding concatenation and minification. 

But that is another issue: https://github.com/mumuki/mumuki-editor/issues/108

**Pull request merged**